### ISSUE TITLE
Game crashing in Manjaro Linux

### ISSUE THREAD

**Issue by egg-that-reads** [Better-than-Adventure/bta-issues#287]
**Describe the bug**

Version: 1.7.7.0_02
Description: I just tried to open the game for the first time in the Arch version of MultiMC 5 (0.7.0-3802), and it fails to start.

**To Reproduce**
1. Open the game In MultiMC
2. The game crashes.

**Expected behavior**
The game opens and plays normally.

**Additional context**
I'm running this mod on Manjaro Linux.

**Crash Logs**

MultiMC version: 0.7.0-3802

Launched instance in online mode

authserver.mojang.com resolves to:
    [13.224.2.2, 13.224.2.21, 13.224.2.98, 13.224.2.107]

session.minecraft.net resolves to:
    [13.107.246.70, 13.107.213.70, 2620:1ec:bdf::70, 2620:1ec:46::70]

textures.minecraft.net resolves to:
    [13.107.213.70, 13.107.246.70, 2620:1ec:46::70, 2620:1ec:bdf::70]

api.mojang.com resolves to:
    [13.107.246.70, 13.107.213.70, 2620:1ec:bdf::70, 2620:1ec:46::70]

Minecraft folder is:
/home/isaac/.local/share/multimc/instances/1.7.7.0_02/.minecraft

Java path is:
/usr/bin/java

Checking Java version...
Java is version 20.0.2, using 64-bit architecture, from N/A.

AMD Ryzen 9 7950X3D 16-Core Processor
Advanced Micro Devices, Inc. [AMD/ATI] Navi 31 [Radeon RX 7900 XT/7900 XTX] (rev c8)

Subsystem: XFX Limited RX-79XMERCB9 [SPEEDSTER MERC 310 RX 7900 XTX]

Kernel driver in use: amdgpu

OpenGL version string: 4.6 (Compatibility Profile) Mesa 23.1.6-3

Main Class:
  

Native path:
  /home/isaac/.local/share/multimc/instances/1.7.7.0_02/natives

Traits:
traits texturepacks
traits legacyLaunch
traits noapplet

Libraries:
  /home/isaac/.local/share/multimc/libraries/net/java/jinput/jinput/2.0.5/jinput-2.0.5.jar
  /home/isaac/.local/share/multimc/libraries/net/java/jutils/jutils/1.0.0/jutils-1.0.0.jar
  /home/isaac/.local/share/multimc/libraries/org/lwjgl/lwjgl/lwjgl/2.9.4-nightly-20150209/lwjgl-2.9.4-nightly-20150209.jar
  /home/isaac/.local/share/multimc/libraries/org/lwjgl/lwjgl/lwjgl_util/2.9.4-nightly-20150209/lwjgl_util-2.9.4-nightly-20150209.jar
  /home/isaac/.local/share/multimc/instances/1.7.7.0_02/.minecraft/bin/minecraft.jar

Native libraries:
  /home/isaac/.local/share/multimc/libraries/net/java/jinput/jinput-platform/2.0.5/jinput-platform-2.0.5-natives-linux.jar
  /home/isaac/.local/share/multimc/libraries/org/lwjgl/lwjgl/lwjgl-platform/2.9.4-nightly-20150209/lwjgl-platform-2.9.4-nightly-20150209-natives-linux.jar

Jar Mods:
  bta (7d59ca7e-ef5c-4151-aee1-7b43174b2eb0.jar)

Params:
  isaac_the_great_ token:<ACCESS TOKEN>:<PROFILE ID> --gameDir /home/isaac/.local/share/multimc/instances/1.7.7.0_02/.minecraft

Window size: 854 x 480

Java Arguments:
[-Dfml.ignoreInvalidMinecraftCertificates=true, -Dfml.ignorePatchDiscrepancies=true, -Xms512m, -Xmx1024m, -Duser.language=en]

Minecraft process ID: 41202

Using onesix launcher.

[00:35:22] [main] 24 Achievements
[00:35:22] [main] 394 recipes
[00:35:22] [Client-Main] Skipping bad option: shaderOverride:
[00:35:22] [Client-Main] Skipping bad option: lastServer:
[00:35:22] [Client-Main] Skipping bad option: selectedController:
[00:35:22] [Client-Main] Resolution: 854 x 480
[00:35:22] [Client-Main] Java Version: 20.0.2
Inconsistency detected by ld.so: dl-lookup.c: 107: check_match: Assertion `version->filename == NULL || ! _dl_name_match_p (version->filename, map)' failed!
Process exited with exit code 127 (0x7f).
Please note that usually neither the exit code, nor its description are enough to diagnose issues!
Always upload the entire log and not just the exit code.
Clipboard copy at: 12 Sep 2023 00:48:36 -0700

**Comment by DianaIsntHere** [comment]
beta minecraft doesnt support java 20, you need java 8

**Comment by forkiesassds** [comment]
> beta minecraft doesnt support java 20, you need java 8

This is wrong, it does. However the Java distribution has issues with LWJGL 2 and newer versions of Java. It doesn't happen with other distributions like Eclipse Adoptium.

**Comment by DianaIsntHere** [comment]
> However the Java distribution has issues with LWJGL 2 and newer versions of Java.

oh, i see what you mean. using java 8 fixed the problem for me so i assumed older versions of minecraft didn't support newer java versions, didn't mean to misinform or anything :)

**Comment by imlighty** [comment]
Honestly, it's more of a you problem: Manjaro Linux is the most messy distribution you could ever install. From stuff such as installing literally W.I.P. versions of stuff to SSL updates delayed, it's a huge mess. If you want a beginner-friendly distribution, I recommend either Linux Mint or Fedora

**Comment by Beethoven-n** [comment]
yeah part of fixing your problem might well be installing another distro. given manjaro's tendencies, you generally have a less stable distro than arch

**Comment by KieranCrossland** [comment]
manjaro is a failure of an OS, they cant even maintain an SSL cert.
